frontend/src/routes/forget/+page.svelte

```diff
@@ -0,0 +1,49 @@
+<script>
+    import '$styles/Auth.pcss';
+    import { enhance } from '$app/forms';
+</script>
+
+<html class="h-full">
+    <div class="auth-page">
+        <div class="container-page">
+        <div class="auth-row">
+            <div class="p-4 sm:p-7">
+            <div class="text-center">
+                <h1 class="auth-title-style">Forgot password?</h1>
+                <p class="auth-subtitle-style">
+                Remember your password?
+                <a class="auth-redirect-button-style" href="/login">
+                    Sign in here
+                </a>
```

Comment:
Seems like a typo here. If a user remembers their password, then they would need to "Login here"